src/EnergyPlus/DXCoils.cc

```diff
@@ -15687,8 +15687,8 @@ void GetDXCoilIndex(EnergyPlusData &state,
                     std::string const &DXCoilName,
                     int &DXCoilIndex,
                     bool &ErrorsFound,
-                    Optional_string_const ThisObjectType,
-                    Optional_bool_const SuppressWarning)
+                    std::string_view const ThisObjectType,
```

Comment:
This one is a good example of the easier changes. The argument is only used for errors, so the argument type is changed first.